app/services/host_inventory_api.rb

```diff
@@ -26,9 +28,29 @@ def inventory_host
     @inventory_host = host_already_in_inventory(@id)
     raise ::InventoryHostNotFound if @inventory_host.blank?
 
+    if (os_release = host_os_releases([@id]).first)
+      @inventory_host.os_major = os_release.os_major
+      @inventory_host.os_minor = os_release.os_minor
+    end
     @inventory_host
   end
 
+  def host_os_releases(ids)
+    response = Platform.connection.get(
+      "#{@url}/#{ids.join(',')}/system_profile", { per_page: 50, page: 1 },
+      X_RH_IDENTITY: @b64_identity
+    )
+    response.body['results'].inject([]) do |acc, host|
+      os_major, os_minor = host['system_profile']['os_release'].split('.')
+      acc << { id: host['id'], os_major: os_major, os_minor: os_minor }
+    end
+  end
+
+  def import_os_releases(ids)
+    os_releases = host_os_releases(ids)
+    Host.import os_releases
+  end
```

Comment:
This looks out of place here, IMO - why should the `HostInventoryApi` know or do anything with our `Host` model?